examples/statistics/confidence_ellipse.py

```diff
@@ -0,0 +1,206 @@
+"""
+======================================================
+Plot a confidence ellipse of a two-dimensional dataset
+======================================================
+
+This example shows how to plot a confidence ellipse of a
+two-dimensional dataset, using its pearson correlation coefficient.
+
+The approach that is used to obtain the correct geometry is
+explained and proved here:
+
+https://carstenschelp.github.io/2018/09/14/Plot_Confidence_Ellipse_001.html
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.patches import Ellipse
+import matplotlib.transforms as transforms
+
+
+def confidence_ellipse(x, y, ax, n_std=3.0, **kwargs):
+    """
+    Create a plot of the covariance confidence ellipse of `x` and `y`
+
+    Parameters
+    ----------
+    x, y : array_like, shape (n, )
+        Input data
+        
+    ax : matplotlib.axes object to the ellipse into
+    
+    n_std : number of standard deviations to determine the ellipse's radiuses
+
+    Returns
+    -------
+    None
+
+    Other parameters
+    ----------------
+    kwargs : `~matplotlib.patches.Patch` properties
+
+    author : Carsten Schelp
+    license: GNU General Public License v3.0 (https://github.com/CarstenSchelp/CarstenSchelp.github.io/blob/master/LICENSE)
```

Comment:
Not a lawyer so pinging @tacaswell on this, but I think GPL won't be acceptable.  Anyone can dismiss once resolved.

Reply:
And most other examples don’t have an author block....

Reply:
> Not a lawyer so pinging @tacaswell on this, but I think GPL won't be acceptable.  Anyone can dismiss once resolved.

When no other example bothers with licenses I won't, either. I just thought that the python license was GPLv3.0 compatible?

Reply:
Oh - matplotlib has a license of its own. Either way  - I removed the license block. I hope that it is ok now if I resolve this conversation?

Reply:
@anntzer: Hi, thanks to the community this is pull request is approaching a state that might be called 'mature'. Do you agree that your change request has been addressed appropriately? It was about that probably problematic license-tag in the docstring.